tools/webassembly.py

```diff
@@ -113,26 +100,21 @@ class Module:
   for parsing the dylink section."""
   def __init__(self, filename):
     with open(filename, 'rb') as f:
-      self.buf = f.read()
-    assert self.buf[:4] == b'\0asm'
-    assert self.buf[4:8] == b'\x01\0\0\0'
-    self.offset = 8
+      self.buf = io.BytesIO(f.read())
```

Comment:
first i see `BytesIO`. what is the benefit?

Reply:
Yeah me to!   Its just way to turn these bytes into a file-like object that many APIs expect.  

As an alternative I think we could just use the open file `f` ... i can do that if you prefer.

Reply:
I like the API, sgtm as is.

Reply:
I updated to avoid this and just used the open file!